backend/fms_core/resources/extraction.py

```diff
@@ -154,7 +156,43 @@ def import_field(self, field, obj, data, is_m2m=False):
                     comment=f"Automatically generated via extraction template import on "
                             f"{datetime.utcnow().isoformat()}Z"
                 )
+            except Exception as e:
+                errors["container"] = ValidationError([f"Could not create destination container. " + e.messages.pop()], code="invalid")
+                
+        try:
+            super().import_obj(obj, data, dry_run)
+        except ValidationError as e:
+            errors.update(e.update_error_dict(errors).copy())
+
+        try:
+            # Create a process for the current extraction
+            self.process = Process.objects.create(protocol=Protocol.objects.get(name="Extraction"),
```

Comment:
Would eventually be better to have constants mapped to the name of protocols, rather than using each time strings like name="Extraction"